ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -762,12 +762,8 @@ def test_chainexpiration_check_without_cert(self, move_ipa_ca_crt):
         Testcase checks that ERROR message is displayed
         when ipa ca crt file is not renamed
         """
-        error_text = (
-            "[Errno 2] No such file or directory: '{}'"
-            .format(paths.IPA_CA_CRT)
-        )
         msg_text = (
-            "Error opening IPA CA chain at {key}: {error}"
+            "Error opening IPA CA chain at '{}'".format(paths.IPA_CA_CRT)
```

Comment:
This line should not be changed upstream, please see https://github.com/freeipa/freeipa-healthcheck/commit/1188d5beaf4a0fdfe98608d9259532b860dabeb1